_config.yml

```diff
@@ -1,4 +1,4 @@
-highlighter: rouge
```

Comment:
The code highlighted by rouge is hardly readable.

Anyway, I switched back to rouge. Something else will hack the css to improve readability.